homeassistant/components/matter/__init__.py

```diff
@@ -242,3 +300,34 @@ async def open_commissioning_window(call: ServiceCall) -> None:
         open_commissioning_window,
         vol.Schema({"device_id": str}),
     )
+
+
+async def async_ensure_addon_running(hass: HomeAssistant, entry: ConfigEntry) -> None:
```

Comment:
I would prefix this method with `_` or else you cannot guarantee it is only run during async_setup_entry and so cannot raise `ConfigEntryNotReady`. Any reason you don't inline this method?

Reply:
The async_setup_entry function of these integrations have a tendency to grow and become hard to read unless we break it down.